pkg/helpers/azureconst_test.go

```diff
@@ -4,10 +4,289 @@
 package helpers
 
 import (
+	"encoding/json"
 	"strings"
 	"testing"
 )
 
+func TestGetAzureLocations(t *testing.T) {
+	expectedLocationMap := map[string]struct{}{
```

Comment:
It's annoying to duplicate these big data structures, but I guess that's correct black box testing.